rllib/connectors/tests/test_agent.py

```diff
@@ -156,6 +159,7 @@ def test_state_buffer_connector(self):
         self.assertEqual(with_buffered[0].data[SampleBatch.ACTIONS], [1, 2, 3])
 
     def test_view_requirement_connector(self):
+        # TODO: @kourosh remove this test when we have a better way to test
```

Comment:
doesn't hurt to keep it? 😭

Reply:
yes but it will be covered in the next test suite that is right now skipped.